opt_out_http_api/api_methods.py

```diff
@@ -45,12 +54,26 @@ def response(self, request, status_code=200, status_reason="OK", **data):
         })
         return json.dumps(data)
 
-# GET Method
+# Error Handling
+
+    @app.handle_errors(OptOutNotFound)
+    def opt_out_not_found(self, request, failure):
+        return self.response(
+            request, status_code=404, status_reason="Opt out not found.")
+
+    @app.handle_errors(OptOutAlreadyExists)
+    def opt_out_already_exists(self, request, failure):
+        return self.response(
+            request, status_code=409, status_reason="opt out already exists.")
+
+
+# Methods
 
     @app.route('/')
     def addresses(self, request):
             request.setHeader('Content-Type', 'application/json')
             return json.dumps(self._info)
+# GET Method
```

Comment:
I would remove the ``# GET Method`` and ``# PUT Method`` comments because they're duplicating things that are already fairly clearly written in the code (i.e. in the ``@app.route(..., methods=['PUT']`` decorator).